.changes/unreleased/tinymce-TINY-11257-2024-09-16.yaml

```
@@ -0,0 +1,7 @@
+project: tinymce
+kind: Fixed
+body: Resize handle would be incorrectly located when no other components were added
+  to status bar.
```

The bar has things on it by default, so the integrator action to trigger the issue was removing things.

```suggestion
body: The editor resize handle was incorrectly rendered when all components were
  removed from the status bar.
```